src/components/Drawer.jsx

```diff
@@ -13,86 +13,82 @@ const sizes = {
-    isOpen = false,
-    title,
-    size = "medium",
-    content,
-    onClose,
-    classNames = {},
-    position = "right",
-}) => {
-    return (
-        <Transition.Root show={isOpen} as={Fragment}>
-            <Dialog
-                as="div"
-                className={clsx("ui-drawer fixed inset-0 z-10 overflow-hidden", classNames.dialog)}
-                open={isOpen}
-                onClose={onClose}
-            >
-                <div className="flex h-screen w-full">
-                    <Transition.Child
-                        as={Fragment}
-                        enter="ease-in-out duration-500"
-                        enterFrom="opacity-0"
-                        enterTo="opacity-100"
-                        leave="ease-in-out duration-500"
-                        leaveFrom="opacity-100"
-                        leaveTo="opacity-0"
-                    >
-                        <Dialog.Overlay className="absolute inset-0 bg-gray bg-opacity-75 transition-opacity" />
-                    </Transition.Child>
-                    <div
-                        className={clsx(
-                            "fixed inset-y-0 flex max-w-full",
-                            position === "right" ? "right-0" : "left-0",
-                            classNames.dialogContent,
-                        )}
-                    >
+export const Drawer = forwardRef(
+    ({ isOpen = false, title, size = "medium", content, onClose, classNames = {}, position = "right" }, ref) => {
+        return (
+            <Transition.Root ref={ref} show={isOpen} as={Fragment}>
+                <Dialog
+                    as="div"
+                    className={clsx("ui-drawer fixed inset-0 z-10 overflow-hidden", classNames.dialog)}
+                    open={isOpen}
+                    onClose={onClose}
+                >
+                    <div className="flex h-screen w-full">
                         <Transition.Child
                             as={Fragment}
-                            enter="transform transition ease-in-out duration-500 sm:duration-700"
-                            enterFrom={position === "right" ? "translate-x-full" : "-translate-x-full"}
-                            enterTo="translate-x-0"
-                            leave="transform transition ease-out-in duration-500 sm:duration-700"
-                            leaveFrom="translate-x-0"
-                            leaveTo={position === "right" ? "translate-x-full" : "-translate-x-full"}
+                            enter="ease-in-out duration-500"
+                            enterFrom="opacity-0"
+                            enterTo="opacity-100"
+                            leave="ease-in-out duration-500"
+                            leaveFrom="opacity-100"
+                            leaveTo="opacity-0"
+                        >
+                            <Dialog.Overlay className="absolute inset-0 bg-black bg-opacity-80 transition-opacity" />
```

Comment:
Can you add a `classNames` prop for this like we have `classNames.content` below.

Also please provide a screenshot of how the updated Drawer looks